From 0d6eef8eeb4124a0598f7c109181f4fe7674484f Mon Sep 17 00:00:00 2001
From: Atul Dhudase <quic_adhudase@quicinc.com>
Date: Tue, 31 Oct 2023 11:30:18 +0530
Subject: [PATCH 2/2] PENDING: arm64: dts: qcom: qcm6490: Add
 gpio-reserved-ranges

Add gpio-reserved-ranges for QCM6490 boards.

Signed-off-by: Atul Dhudase <quic_adhudase@quicinc.com>
Upstream-Status: Pending
---
 arch/arm64/boot/dts/qcom/qcm6490.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/qcm6490.dtsi b/arch/arm64/boot/dts/qcom/qcm6490.dtsi
index cccb50ce6269..e05e0f3b4b12 100644
--- a/arch/arm64/boot/dts/qcom/qcm6490.dtsi
+++ b/arch/arm64/boot/dts/qcom/qcm6490.dtsi
@@ -129,6 +129,10 @@ &lpass_hm {
 	status = "disabled";
 };
 
+&tlmm {
+	gpio-reserved-ranges = <32 2>, <48 4>;
+};
+
 &video_mem {
 	reg = <0x0 0x8a700000 0x0 0x500000>;
 };
-- 
2.25.1

